tests/nodeos_high_transaction_test.py

```diff
@@ -185,9 +185,9 @@ def cacheTransIdInBlock(transId, transToBlock, node):
                 else:
                     if Utils.Debug:
                         Print("Transaction not found for trans id: %s. Will wait %d seconds to see if it arrives in a block." %
-                              (transId, args.transaction_time_delta))
+                              (transId, args.transaction_time_delta + 10))
                     transTimeDelayed = True
-                    node.waitForTransInBlock(transId, timeout = args.transaction_time_delta)
+                    node.waitForTransInBlock(transId, timeout = args.transaction_time_delta + 10)
```

Comment:
Instead of adding 10 here, change `--transaction-time-delta` default from 5 to 15.